### PR TITLE
Adding New fields + fields from Kong OSS PR  8841

### PR DESCRIPTION
**Pull request by jeremyjpj0916**
Related: https://github.com/Kong/kong/pull/8841

### CODE REVIEW

File: app/_hub/kong-inc/request-termination/2.0.x.md [hunk 1]
```diff
@@ -69,17 +69,42 @@ params:
...
     sending the configured response.
 
 ---
 
-## New in 2.0.0
+## New in 2.2.0
 
-There were changes in the Plugin Handler structure and on the Plugins DAO (`load_plugin_schemas`) that make this plugin
-backwards-incompatible if another plugin depended on it.
+Addition of the avert option, which prevents plugin execution when matched. Opposite from the trigger option
```
Comment (Guaris) [inline app/_hub/kong-inc/request-termination/2.0.x.md:106]:
```suggestion
Added the `avert` option, this prevents plugin execution when matched. `avert` is like the opposite of the `trigger` option 
```

**Review by Guaris — CHANGES_REQUESTED** [review]

**Comment by jeremyjpj0916** [comment]
Agree on the word smithing.

**Comment by Guaris** [comment]
> Agree on the word smithing.

This PR depends on this one right? https://github.com/Kong/kong/pull/8841

**Comment by jeremyjpj0916** [comment]
@Guaris Correct. Unit tests failing for odd reason, one error was an unrelated postgres and the other error idk how my SRC changes could have even caused that failure after studying it a few times.

**Comment by Guaris** [comment]
If you could ping me when that other PR gets merged. I'll merge this one then.

**Comment by Guaris** [comment]
I'm closing this PR on account of the other PR being closed 
https://github.com/Kong/kong/pull/8841

please reopen this PR when you are ready!

**Pull request closed without merging**